feat: support tokens scoped to multiple repositories within organization (#46)
actions/create-github-app-token: README.md

1. [Register new GitHub App](https://docs.github.com/apps/creating-github-apps/setting-up-a-github-app/creating-a-github-app)
2. [Store the App's ID in your repository environment variables](https://docs.github.com/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows) (example: `APP_ID`)
3. [Store the App's private key in your repository secrets](https://docs.github.com/actions/security-guides/encrypted-secrets?tool=webui#creating-encrypted-secrets-for-a-repository) (example: `PRIVATE_KEY`)

> 4. We recommend that `owner` be set to `env.GITHUB_REPOSITORY_OWNER` (See [environment variables](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables))

Reviewer: I don't think I would recommend this in general. I would recommend it when describing the `owner` and `repositories` arguments. We can add another usage example for a token that has access to all repositories in the current repository owner's installation

Author: Removed in favor of use cases.

> ### Minimal usage

Reviewer: We can rename it to make clear what it dose

```suggestion
### Create token for current repository
```

I would then add more usage examples

1. Create token for all repositories in current owner's installation
2. Create a token for selected repositories in current owner's installation
3. Create token for all repositories in another owner's installation

Note that an owner can be both an organization or a user

Author: Ah gotcha, that makes sense. Done!

        with:
          app_id: ${{ vars.APP_ID }}
          private_key: ${{ secrets.PRIVATE_KEY }}
          owener: ${{ github.repository_owner }}

>           repositories: repo1,repo2

Reviewer: remove from this usage example, see above

Author: Done.

          # required
          app_id: ${{ vars.APP_ID }}
          private_key: ${{ secrets.PRIVATE_KEY }}
          # optional - if not used, defaults to current repository owner
          owner: ${{ github.repository_owner }}
          # optional - if not used, defaults to all repositories in the organization where the app is installed

>           repositories: repo1,repo2

Reviewer: remove from this usage example, I'd focus on the use case

Author: Done.

**Required:** GitHub App private key.

### `owner`

**Optional:** GitHub App installation owner. If empty, defaults to the current repository owner.

### `repositories`

> **Optional:** Comma-separated list of repositories to grant access to. If 'owner' is set and 'repositories' is empty then the access will be scoped to all repositories in the organization where the app is installed. If 'owner' and 'repositories' are empty then the access will be scoped to the current repository.

Reviewer: ```suggestion
**Optional:** Comma-separated list of repositories to grant access to. If 'owner' is set and 'repositories' is empty then the access will be scoped to all repositories in the current repository owner's installation. If 'owner' and 'repositories' are empty then the access will be scoped to the current repository.
```

Author: Done.

**Required:** GitHub App private key.

### `owner`

**Optional:** GitHub App installation owner. If empty, defaults to the current repository owner.

### `repositories`

> **Optional:** Comma-separated list of repositories to grant access to. If 'owner' is set and 'repositories' is empty then the access will be scoped to all repositories in the current repository owner's installation. If 'owner' and 'repositories' are empty then the access will be scoped to the current repository.

Reviewer: ```suggestion
**Optional:** Comma-separated list of repositories to grant access to. If 'owner' is set and 'repositories' is empty then the access will be scoped to all repositories in the provided repository owner's installation. If 'owner' and 'repositories' are empty then the access will be scoped to only the current repository.
```
